package.json

```diff
@@ -61,34 +61,34 @@
     "@bbc/psammead-amp-geo": "^1.1.2",
     "@bbc/psammead-assets": "^2.7.0",
     "@bbc/psammead-brand": "^5.0.11",
-    "@bbc/psammead-calendars": "^2.0.1",
+    "@bbc/psammead-calendars": "^2.0.2",
     "@bbc/psammead-caption": "^2.2.18",
     "@bbc/psammead-consent-banner": "^2.3.20",
     "@bbc/psammead-copyright": "^1.2.16",
     "@bbc/psammead-figure": "^1.2.8",
-    "@bbc/psammead-headings": "^3.1.18",
+    "@bbc/psammead-headings": "^3.1.19",
     "@bbc/psammead-image": "^1.2.3",
     "@bbc/psammead-image-placeholder": "^1.2.21",
     "@bbc/psammead-inline-link": "^1.3.15",
     "@bbc/psammead-locales": "^3.0.1",
     "@bbc/psammead-media-indicator": "^2.6.11",
-    "@bbc/psammead-media-player": "^2.2.0",
+    "@bbc/psammead-media-player": "^2.2.1",
     "@bbc/psammead-navigation": "^3.0.5",
     "@bbc/psammead-paragraph": "^2.2.18",
-    "@bbc/psammead-rich-text-transforms": "^1.1.2",
+    "@bbc/psammead-rich-text-transforms": "^2.0.0",
```

Comment:
can you double check with the media pod that we are good to pull in this new major version please

Reply:
> Last update was breaking, but it's only being used on a page that isn't even live yet anyway so no real risk there.